src/collective/collectionfilter/filteritems.py

```diff
@@ -213,6 +217,9 @@ def get_filter_items(
                 or filter_value in value_blacklist
             ):
                 continue
+            if isinstance(filter_value, int):
```

Comment:
This breaks boolean `filter_value` because `isinstance(<False or True>, int) == True` and therefore boolean values get converted to strings

Reply:
https://stackoverflow.com/questions/37888620/comparing-boolean-and-int-using-isinstance 😉

Reply:
yay.

Reply:
I fixed that ... and portlets too with the new `reverse` setting